### DISCUSSION TITLE
https://huyvu15.github.io/posts/arbdv92f?fbclid=IwY2xjawEvARtleHRuA2FlbQIxMQABHfXoPIy5lZLFjphWxO0Eh1u6n2VLTnmu2fyy2rWxUC2BKdtkP0-pO3V-ZQ_aem_77gfWWgf9ZnAMma2eToxHA

### DISCUSSION THREAD

**Discussion by giscus** [huyvu15/huyvu15.github.io#59] [Show and tell]
# https://huyvu15.github.io/posts/arbdv92f?fbclid=IwY2xjawEvARtleHRuA2FlbQIxMQABHfXoPIy5lZLFjphWxO0Eh1u6n2VLTnmu2fyy2rWxUC2BKdtkP0-pO3V-ZQ_aem_77gfWWgf9ZnAMma2eToxHA

Thoạt đầu Cho những ai hay đọc blogs của mình thì nay mình lại ngoi lên để chia sẻ một số kiến thức kinh nghiệm hiện có. Xin chào, mình là GluTis một sinh viên ngành toán tại Hust. Trong thời gian đi

https://huyvu15.github.io/posts/arbdv92f?fbclid=IwY2xjawEvARtleHRuA2FlbQIxMQABHfXoPIy5lZLFjphWxO0Eh1u6n2VLTnmu2fyy2rWxUC2BKdtkP0-pO3V-ZQ_aem_77gfWWgf9ZnAMma2eToxHA

**Comment by huyvu15** [comment]
a
